lib/interpret_and_optimize/entities/layouts/auto_layout_align_strategy.dart

```diff
@@ -90,6 +90,27 @@ class AutoLayoutAlignStrategy extends AlignStrategy<PBLayoutIntermediateNode> {
   /// and adds a container on top of it
   PBIntermediateNode _needsContainer(
       PBIntermediateNode child, bool isVertical, PBContext context) {
+    /// Special case for InheritedBitmap
+    /// Check parent constraints and if it is not stretch, set them to
+    /// CENTER defaults
+    /// TODO: Improve if statements
+    if (child is InheritedBitmap) {
+      if (isVertical) {
+        if (child.layoutMainAxisSizing == ParentLayoutSizing.INHERIT) {
```

Comment:
We could get rid of this if statements by writing something like `child.constraints.fixedHeight = child.layoutMainAxisSizing == ParentLayoutSizing.INHERIT`